### PR TITLE
Update Capsule Modal

### PR DESCRIPTION
**Pull request by vr2amesh**
### Checklist
- [ ] Increment the version field in `package.json` of the package you have made changes in following [semantic versioning](https://semver.org/) and using alpha release tagging
- [ ] Check the box that allows repo maintainers to update this PR
- [ ] Test locally to make sure this feature/fix works
- [ ] Run `yarn check-all` to confirm there are not any associated errors
- [ ] Confirm this PR passes Circle CI checks
- [ ] Add or update relevant information in the documentation

### Docs Checklist
- [ ] Include a screenshot of any changes ([see docs README on running locally](https://github.com/blocknative/web3-onboard/blob/develop/docs/README.md))
- [ ] Add/update the appropriate package README (if applicable)
- [ ] Add/update the related module in the [docs demo](https://github.com/blocknative/web3-onboard/blob/develop/docs/src/lib/services/onboard.js) (if applicable)
- [ ] Add/update the related package in the `docs/package.json` file (if applicable)

### If this PR includes changes to add an injected wallet or SDK wallet module: 
Please complete the following using the internal demo package.
To run this demo use the command `yarn && yarn dev` to get the project running at `http://localhost:8080/`

#### Tests with demo app (injected)
- [ ] send transaction
- [ ] switch chains
- [ ] sign message
- [ ] sign typed message
- [ ] disconnect

#### Tests with demo app (SDK)
- [ ] send transaction
- [ ] switch chains
- [ ] sign message
- [ ] sign typed message
- [ ] disconnect

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -55,7 +56,7 @@
   "scripts": {
     "install-m1-mac": "yarn install --ignore-optional",
     "format": "prettier --write 'packages/**/*.ts'",
-    "dev": "yarn wsrun dev",
+    "dev": "PORT=8081 yarn wsrun dev",
```
Comment (vr2amesh) [inline package.json:59]: will remove this after review

File: packages/capsule/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/capsule",
-  "version": "2.0.1",
+  "version": "2.0.1-alpha.1",
```
Comment (Adamj1232) [inline packages/capsule/package.json:3]:
```suggestion
  "version": "2.0.2-alpha.1",
```

File: packages/capsule/yarn.lock [hunk 3]
```diff
@@ -7,61 +7,52 @@
   resolved "https://registry.yarnpkg.com/@adraffy/ens-normalize/-/ens-normalize-1.10.0.tgz#d2a39395c587e092d77cbbc80acf956a54f38bf7"
   integrity sha512-nA9XHtlAkYfJxY7bce8DcN7eKxWWCWkU+1GR9d+U6MbNpfwQp8TI7vqOsBsMcHoT4mBu2kypKoSKnghEzOOq5Q==
```
Comment (Adamj1232) [inline packages/capsule/yarn.lock:8]: yarn.lock at the package level can be removed

File: packages/capsule/package.json (hunk 2 above)
Comment (Adamj1232) [reply packages/capsule/package.json:3]: Please also bump this version in the packages/demo and confirm everything works from the internal demo. Will want to update this version in the docs site too docs/package.json

**Comment by leightkt** [comment]
@vr2amesh I was able to connect a capsule wallet to the demo, but not complete any of the checklist functions listed:

**Comment by vr2amesh** [comment]
> @vr2amesh I was able to connect a capsule wallet to the demo, but not complete any of the checklist functions listed: send transaction switch chains sign message sign typed message
> 
> can you take a look?

Will take a look now

**Comment by leightkt** [comment]
@vr2amesh I connect my wallet and can switch chains- but when I click sign transaction or send transaction- nothing happnes

**Comment by leightkt** [comment]
@vr2amesh ok so I see a console log of the signed message- I was expecting some sort of UI with Capsule. 
I was also able to confirm sending a transaction- again, is there no UI for these functions? (just want to make sure I'm understanding correctly).

File: packages/capsule/yarn.lock (hunk 3 above)
Comment (leightkt) [reply packages/capsule/yarn.lock:8]: @vr2amesh please remove the capsule/yarn.lock file

**Comment by leightkt** [comment]
@vr2amesh got ya- thank you for confirming and sorry for the confusion! overall- this PR is looking good. Can you please remove the capsule/yarn.lock file, the update to PORT in package.json and merge in the most recent web3onboard develop branch? I can then get this approved/merged.

File: package.json [hunk 4]
```diff
@@ -7,8 +7,6 @@
       "./packages/*"
     ],
     "nohoist": [
-      "**/react",
```
Comment (leightkt) [inline package.json:10]: @vr2amesh can you undo these changes to package.json to see if the build completes successfully?

Comment (leightkt) [reply package.json:10]: @vr2amesh I'm seeing this error when running `yarn run build` locally in this repo: 
`| ERROR in ../../node_modules/@toruslabs/base-controllers/node_modules/@ethereumjs/util/dist/ssz.js 4:14-39
 | Module not found: Error: Can't resolve '@chainsafe/ssz' in '/Users/katleight/Blocknative/web3-onboard-capsule/node_modules/@toruslabs/base-controllers/node_modules/@ethereumjs/util/dist'`

Comment (vr2amesh) [reply package.json:10]: @leightkt I think there are some issues with the merge I performed. I tried this on a fresh branch and things seem to be in order. Let me recreate the branch shortly and then will ping you right away here

Comment (vr2amesh) [reply package.json:10]: @leightkt Updated the yarn.lock file to match a proper fresh branch. Looks like we've got the build issues fixed

Comment (leightkt) [reply package.json:10]: looking good! thanks for all your hard work on this!

**Review by leightkt — APPROVED** [review]

File: package.json (hunk 4 above)
Comment (vr2amesh) [reply package.json:10]: @leightkt Thank you for the quickness in response. It is sincerely appreciated :)
